build/containers/Dockerfile.anomaly-detection-app

```diff
@@ -0,0 +1,27 @@
+ARG PLATFORM=amd64
+FROM ${PLATFORM}/debian:buster-slim
+
+# Avoid tzdata prompt 
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN echo "Creating container based on ${PLATFORM}/debian:buster-slim" && \
+    apt update && \
+    apt install -y protobuf-compiler libprotoc-dev python3-pip \
+    python3-grpcio python3-sklearn && \
+    apt clean
+
+RUN pip3 install --upgrade pip && \
+    pip3 install numpy protobuf flask
+
+RUN mkdir app
```

Comment:
Any specific reason why these three RUNs are not a single RUN?

Reply:
Not a good one. Will fix thanks!